base-notebook/start.sh

```diff
@@ -104,6 +105,23 @@ if [ "$(id -u)" == 0 ] ; then
         useradd --home "/home/${NB_USER}" --uid "${NB_UID}" --gid "${NB_GID}" --groups 100 --no-log-init "${NB_USER}"
     fi
 
+    # Create additional groups for NB_USER
+    _IFS=${IFS}; IFS=,
+    for gid in ${NB_UNPRIVILEGED_GROUPS?}; do
+        g_id=$(echo "${gid}" | awk -F '[()]' '{print $1}')
+        g_name=$(echo "${gid}" | awk -F '[()]' '{print $2}')
+        # make sure group name, no longer than 32 characters
+        g_name="${g_name:0:32}"
```

Comment:
I don't think we should manipulate user's input this way.
If they use long group name and it fails when starting the container, it's actually better, than fixing the group name and making them believe everything is fine.